src/drawer.multicanvas.js

```diff
@@ -390,11 +390,13 @@ export default class MultiCanvas extends Drawer {
      * @param {number} start The x-offset of the beginning of the area that
      * should be rendered
      * @param {number} end The x-offset of the end of the area that
+     * @param {channelIndex} channelIndex The channel index of the line drawn
      * should be rendered
      */
```

Comment:
the "should be rendered" belongs to doc of `end` parameter

Reply:
done